test/backup.ts

```diff
@@ -322,6 +323,114 @@ describe('Backup', () => {
       );
     });
 
+    describe('copy', () => {
+      const INSTANCE_NAME = 'instance-name';
+      const SRC_BACKUP = 'src-backup';
+      const DST_BACKUP = 'dst-backup';
+
+      beforeEach(() => {
+        fakeCodec.encode = util.noop;
+        backup = new Backup(INSTANCE, DST_BACKUP, SRC_BACKUP);
+      });
+
+      it('should succeed with basic params', done => {
+        const timestamp = new google.protobuf.Timestamp();
+        timestamp.seconds = new Date(BACKUP_EXPIRE_TIME).getSeconds();
+
+        const expectedReqOpts = extend(
+          {},
+          {
+            parent: INSTANCE_NAME,
+            backupId: DST_BACKUP,
+            sourceBackup: SRC_BACKUP,
+            expireTime: timestamp,
+          }
+        );
+
+        backup.request = config => {
+          assert.strictEqual(config.client, 'DatabaseAdminClient');
+          assert.strictEqual(config.method, 'copyBackup');
+          assert.deepStrictEqual(config.reqOpts, expectedReqOpts);
+          assert.deepStrictEqual(config.headers, backup.resourceHeader_);
+          done();
+        };
+
+        backup.create(
+          {
+            parent: INSTANCE_NAME,
+            backupId: DST_BACKUP,
+            sourceBackup: SRC_BACKUP,
+            expireTime: timestamp,
```

Comment:
Can't we use `BACKUP_EXPIRE_TIME` directly here, like we are doing in create backup tests ?

Reply:
Got it. Thanks